anti-evil-maid/README

```diff
@@ -191,5 +247,38 @@ and enter your disk decryption passphrase even though you can't see your
 secret. It will be resealed again automatically later in the boot process
 (see step 5.a).
 
+
+What to do in case of compromise
+================================
+
 For a discussion of potential attacks against Anti Evil Maid, see the article
 referenced at the beginning.
+
+
+AEM media copied/stolen by an attacker
+--------------------------------------
+
+If you have your system up and running or have an extra AEM media using which
+you can boot the system before the attacker can get to it:
+
+* `sudo su` in dom0
+* `. /usr/sbin/anti-evil-maid-lib` (note the dot at the beginning)
+* `revokefreshness <suffix>` (where `<suffix>` is the missing label suffix)
+
+In case you do not remember the used media label suffix, they're all stored
+in `/var/lib/anti-evil-maid/tpm-freshness-db`. If the particular media had
+no special suffix set, use `revokefreshness ""`. Alternatively, `resetfreshness`
+will wipe all freshness tokens from the TPM (thus invalidating all enrolled AEM
+media and forcing you to boot an unverified system).
+
+As a last resort, you can attempt to reset the TPM from BIOS (with similar effect
+to `resetfreshness`). Otherwise, it's game over.
+
+
+Someone saw my AEM text secret/LUKS passphrase
+----------------------------------------------
+
+You should've installed AEM in multi-factor mode. Better luck next time.
```

Comment:
Users can at least add a new LUKS passphrase, revoke the old key slots, and reinstall AEM (with a different static text secret if needed). It's not a panacea for SSDs, but still.